paasta_tools/chronos_tools.py

```diff
@@ -505,7 +517,8 @@ def validate(self):
         error_msgs.extend(super(ChronosJobConfig, self).validate())
 
         for param in ['epsilon', 'retries', 'cpus', 'mem', 'disk',
-                      'schedule', 'scheduleTimeZone', 'parents', 'cmd']:
+                      'schedule', 'scheduleTimeZone', 'parents', 'cmd',
```

Comment:
It kinda suck that this validate has to duplicate stuff. Can you put the security and deps in the InstanceConfig object so they are shared for all new types of things? (and not just chronos?)